boards/raspberrypi/rpi_pico2/Kconfig.rpi_pico2

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Andrew Featherstone
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_RPI_PICO2
+	select SOC_RP2350A
```

Comment:
```suggestion
```

Reply:
Remove series selection, should be selected by the core

Reply:
Not addressed?